R/utils.R

```diff
@@ -30,3 +30,32 @@ get_filetoken <- function(filepath = rlang::missing_arg(), content = rlang::miss
   }
   return(token)
 }
+
+#' Checks if the given response was successful or not.
+#'
+#' The function always returns a list with the response's ID. In case of an
+#' error, the list also contains the errors reason. In case of a successful
+#' response, the list also contains the complete response.
+#'
+#' @param res The response to handle
+#'
+#' @return A list with either the error's reason or the complete response
```

Comment:
Maybe add the name of the list elements? as we have no structures to back them for the moment?